src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -234,9 +228,7 @@ void BulkController::send(QList<int> data, unsigned int length) {
     Q_UNUSED(length);
     QByteArray temp;
 
-    foreach (int datum, data) {
-        temp.append(datum);
-    }
+    foreach (int datum, data) { temp.append(datum); }
```

Comment:
Why did it collapse this when it expands most everything else? (e.g. controllers/controller.cpp
line 115 below) Which do we prefer?